core/includes/drupal.inc

```diff
@@ -2592,6 +2592,51 @@ function views_clean_css_identifier($identifier, $filter = array(' ' => '-', '/'
   backdrop_clean_css_identifier($identifier, $filter);
 }
 
+/**
+ * Finds the most recently changed nodes that are available to the current user.
+ *
```

Comment:
Should we include @deprecated in the docblock?

Reply:
@jenlampton or @quicksketch Like this?

![image](https://github.com/user-attachments/assets/b811796d-1b7d-4ed7-85d5-edb31cac8a90)

Reply:
Yes, I think so?

Reply:
That's right but usually `@deprecated` is at the bottom of the docblock. I moved it in a follow-up commit.